### DISCUSSION TITLE
MIT license question

### DISCUSSION THREAD

**Discussion by TexDex** [community/community#59558] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I have a question about MIT license.
If some software is open source,discontinued and licensed under MIT license,am I allowed to create my own commercial CLOSED source software out of that open sourced software and sell it commercially to customers by selling them licenses?(I would basically OWN that software and sell licenses for it to the market)If this is possible,under what conditions would it be possible?

**Comment by leonden** [comment]
Yes, if a software project is licensed under the MIT license and you obtain a copy of that software, you are generally allowed to create your own commercial closed-source software based on it and sell it commercially. Hope that helps 😄
